### PR TITLE
Update settlement timeouts

### PR DESCRIPTION
**Pull request by palango**
Update the settlement timeouts to the decided values.

Part of raiden-network/spec#16

### CODE REVIEW

**Comment by loredanacirstea** [comment]
The problem here is that we need to add at least the minimum settlement timeout as an argument for the `TokenNetworkRegistry` and `TokenNetwork` - otherwise settlement tests will take way too long.

**Comment by palango** [comment]
> The problem here is that we need to add at least the minimum settlement timeout as an argument for the TokenNetworkRegistry and TokenNetwork - otherwise settlement tests will take way too long.

Because we'll have to mine 500 blocks for each test?

**Comment by loredanacirstea** [comment]
> Because we'll have to mine 500 blocks for each test?

Yes. For the settlement window.
So for now, we have decided to set this variable at deployment time.

File: raiden_contracts/constants.py [hunk 1]
```diff
@@ -29,8 +29,11 @@
 EVENT_ADDRESS_REGISTERED = 'AddressRegistered'
 
 # Timeouts
-SETTLE_TIMEOUT_MIN = 6
-SETTLE_TIMEOUT_MAX = 2700000
+TEST_SETTLE_TIMEOUT_MIN = 5
+TEST_SETTLE_TIMEOUT_MAX = 100000
```
Comment (palango) [inline raiden_contracts/constants.py]: Just pulled the values out of my ass, we can revert to the old ones if they have meaning.

**Pull request merged**